### PR TITLE
chore: add fees to send confirmation

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Adds the line item for local amount display of fees on the send confirmation screen.

| Before iOS stCelo | Before iOS ETH | After iOS stCELO | After iOS ETH |
| ---- | ---- | ---- | ---- |
|  |  |  |  |

### Test plan

- Tested locally on iOS
- Tested locally on Android
- Unit test snapshot updated

### Related issues

- Fixes ACT-1135

### Backwards compatibility

Yes

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/send/SendConfirmation.tsx [hunk 1]
```diff
@@ -373,6 +388,13 @@ const styles = StyleSheet.create({
     color: colors.infoDark,
     paddingRight: 8,
   },
+  subHeading: {
+    marginVertical: 0,
+  },
+  subHeadingText: {
+    ...fontStyles.small,
```
Comment (MuckT) [inline src/send/SendConfirmation.tsx:395]: Used deprecated fontStyles as mock aren't updated and to match subHeading of line below.

File: src/send/SendConfirmation.tsx [hunk 2]
```diff
@@ -152,6 +152,7 @@ function SendConfirmation(props: Props) {
         <LineItemRow
           testID="SendConfirmation/fee"
           title={t('feeEstimate')}
+          textStyle={{ ...typeScale.bodyMedium }}
```
Comment (finnian0826) [inline src/send/SendConfirmation.tsx:155]: Why is this being added? Is it different from what it was defaulting to or just ensuring that it is this?

File: src/send/SendConfirmation.tsx (hunk 1 above)
Comment (finnian0826) [reply src/send/SendConfirmation.tsx:395]: Do we want to add a clean up note here or check with Nitya about changing the mocks? I worry that if we add the deprecated style here in the future when we change this screen we'll be in the same boat.

**Review by finnian0826 — APPROVED** [review]
Left a couple questions but otherwise lgtm

File: src/send/SendConfirmation.tsx (hunk 2 above)
Comment (MuckT) [reply src/send/SendConfirmation.tsx:155]: It defaults to `fontStyles.regular`. This was closer to the mocks in regards to font size.

File: src/send/SendConfirmation.tsx (hunk 1 above)
Comment (MuckT) [reply src/send/SendConfirmation.tsx:395]: @finnian0826 unfortunately It's already used in several other places throughout `src/send/SendConfirmation.tsx`; however, I do think it's worthwhile to call out uses of the old fonts on new screens and new mocks.

File: src/send/SendConfirmation.tsx (hunk 2 above)
Comment (satish-ravi) [reply src/send/SendConfirmation.tsx:155]: does this need a color too? I don't think the new styles default a color like the old ones.

File: src/send/SendConfirmation.tsx (hunk 1 above)
Comment (satish-ravi) [reply src/send/SendConfirmation.tsx:395]: is there no equivalent style in typescale? Or is the concern that there's multiple other uses and hard to update now?

File: src/send/SendConfirmation.tsx (hunk 2 above)
Comment (MuckT) [reply src/send/SendConfirmation.tsx:155]: `src/components/LineItemRow.tsx` has a default text color set to `Colors.black` and the subtext is set to `colors.gray4`.

File: src/send/SendConfirmation.tsx (hunk 1 above)
Comment (MuckT) [reply src/send/SendConfirmation.tsx:395]: My main concern is that there are several usages of `fontStyle` within the file. I'll switch them over to typeScale and see if I can get a similar appearance.

Comment (MuckT) [reply src/send/SendConfirmation.tsx]: Was able to update the styles in 55bcde12049b9f9b5fd74fe7e4c6bcdc5f3b2ee1! The snapshots were pretty helpful in the case to make sure the new fonts were as close to the old ones as possible.

**Pull request merged**